perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -388,6 +388,14 @@ sub munge_contacts_to_bodies {
     FixMyStreet::Cobrand::Buckinghamshire::munge_contacts_to_bodies($self, $contacts, $report);
 }
 
+sub post_report_sent {
+    my ($self, $report) = @_;
+
+    # Run Buckinghamshire-specific code
+    my $bucks = FixMyStreet::Cobrand::Buckinghamshire->new;
```

Comment:
Needed to create a new cobrand object here, as calling it directly with the FMS cobrand, it wouldn't have access to _parish_ids.